charts/osm/templates/osm-deployment.yaml

```diff
@@ -73,6 +73,33 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.name
+      {{- if .Values.OpenServiceMesh.enableFluentbit}}
+        - name: fluentbit-logger
+          image: fluent/fluent-bit:1.5
+          imagePullPolicy: Always
+          volumeMounts:
+          - name: config
+            mountPath: /fluent-bit/etc
+            readOnly: true 
+          - name: var-log-containers
+            mountPath: /var/log/containers
+            readOnly: true
+          - name: var-log-pods
+            mountPath: /var/log/pods
```

Comment:
What component is using this volume?

Reply:
Fluent Bit uses these to get access to the osm-controller logs that by default are stored in var/log/containers/. I'm not totally sure why var/log/pods is needed but it does not find any logs without it

Reply:
Ah that makes sense. I did a quick search and found `Kubernetes is using symbolic links from /var/log/containers to /var/log/pods` ref: https://github.com/kubernetes/kubernetes/issues/53022

Reply:
oh that explains why some fluentbit examples had var/lib/containers instead! Can I mark this as resolved?